src/Differ/SebastianBergmannShortDiffer.php

```diff
@@ -24,9 +24,9 @@
      */
     private $differ;
 
-    public function __construct()
+    public function __construct(Differ $differ = null)
```

Comment:
With injecting you killed the difference between `SebastianBergmannDiffer` and `SebastianBergmannShortDiffer`, as one could inject anything violating the idea behind the class. They are suppose to provide diff in given format, not just to proxy things ;)